### PR TITLE
Stats: Add new personal plan purchase page

### PR DESCRIPTION
**Pull request by grzegorz-cp**
Related to #81257

## Proposed Changes

* add a new page for purchasing personal plan (simplified view that doesn't use a purchase wizard component like the previous iterations) - based on changes introduced in #81250

## Testing Instructions

* navigate to the live branch
* **without any feature flag** applied verify that the experience didn't change for:
   * a page without any plan purchased (you should see the old wizard)
   * a page with the commercial plan purchased (you should see a confirmation notice)
   * a page with free or PWYW plan purchased (you should see the old wizard)
* apply `stats/type-detection` feature flag and check the updated experience
  * if the page **wasn't classified as either personal or commercial**, you should see the old wizard
  * if your page **has been classified as personal** you should see a new purchase page for only the personal plan (not the wizard) - this can be tested locally by forcing commercial detection in the controller (`isCommercial` option)
  * if you purchased free or PWYW plans you should see a confirmation full-page notice when visiting the purchase page again

SCR-20230904-kdoq

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

**Review by kangzj — APPROVED** [review]
Works as described and looks good! Thanks for fixing this 👍

File: client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx [hunk 1]
```diff
@@ -60,6 +95,84 @@ const StatsCommercialPurchase = ( {
...
+	siteSlug,
+	maxSliderPrice,
+	pwywProduct,
+	redirectUri,
+	from,
+	adminUrl,
+	disableFreeProduct = false,
+}: StatsPersonalPurchaseProps ) => {
+	const translate = useTranslate();
+	const sliderStepPrice = pwywProduct.cost / MIN_STEP_SPLITS;
```
Comment (kangzj) [inline client/my-sites/stats/stats-purchase/stats-purchase-single-item.tsx:108]: We could take a look later whether we could reduce some of the code duplication here. Definitely not a blocker. 😃

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8861813

Thank you @grzegorz-cp for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8861813).

**Pull request merged**